### DISCUSSION TITLE
PKCE Spotify.Web.Auth

### DISCUSSION THREAD

**Discussion by Alexis-Grimm** [JohnnyCrazy/SpotifyAPI-NET#617] [Q&A] [answered]
As far as I understand in order to get the code for PKCE Authorization you need an embedded web server to return it from the redirect. I'm having issues figuring out how to set one up and I think it was mentioned at some point that you can use Spotify.Web.Auth for this purpose, but there doesn't seem to be any information on its use under PKCE and the example that was mentioned in another discussion (CLI-Persistent Config) is blank.

**Comment by JohnnyCrazy** [comment, marked as answer]
Have a look at https://github.com/JohnnyCrazy/SpotifyAPI-NET/tree/master/SpotifyAPI.Web.Examples/Example.CLI.PersistentConfig, it implements PKCE with `SpotifyAPI.Web.Auth`
